tests/cli/cli_test.go

```diff
@@ -110,27 +104,24 @@ func runTestSpec(t *testing.T, test *TestSpec) {
 			// perform templating on cmdString
 			cmdTmplOutput, tmplErr := templating(strings.Join(cmdString, " "), cache)
 			if tmplErr != nil {
-				t.Log("Executing templating failed: %s. Error: %v", cmdString, tmplErr)
-				t.Fatal()
+				t.Fatal("Executing templating failed: %s. Error: %v", cmdString, tmplErr)
 			}
 			cmdTmplString = strings.Fields(cmdTmplOutput)
 
 			// execute command
 			cmdOutput, cmdErr := exec.Command(cmdTmplString[0], cmdTmplString[1:]...).CombinedOutput()
-			t.Logf("Running: %s", strings.Join(cmdTmplString, " "))
 
 			//perform templating on RegEx string
 			regexTmplOutput, tmplErr := templating(regexMap[cmdSpec.Expectation], cache)
 			if tmplErr != nil {
-				t.Log("Executing templating failed: %s. Error: %v", cmdSpec.Expectation, tmplErr)
-				t.Fatal()
+				t.Fatal("Executing templating failed: %s. Error: %v", cmdSpec.Expectation, tmplErr)
 			}
 
 			// check if the command output matches the RegEx
 			expectedOutput := regexp.MustCompile(regexTmplOutput)
 			if !expectedOutput.MatchString(string(cmdOutput)) {
-				err = fmt.Errorf("Mismatched expected output: %s : Error: %v", cmdOutput, cmdErr)
-				t.Log(err)
+				errString := "Mismatched expected output: " + string(cmdOutput) + ": Error: " + cmdErr.Error()
+ 				err = errors.New(errString)
```

Comment:
white space spotted :) (your editor probably have a Go formatting plugin for that)